tasks/eva_3623/src/main/groovy/eva3623/CorrectSSOperationFromEVA3399.groovy

```diff
@@ -0,0 +1,58 @@
+package eva3623
+
+import org.slf4j.LoggerFactory
+import org.springframework.data.mongodb.core.BulkOperations
+import org.springframework.data.mongodb.core.query.Update
+import uk.ac.ebi.eva.accession.core.model.eva.SubmittedVariantOperationEntity
+import uk.ac.ebi.eva.groovy.commons.EVADatabaseEnvironment
+import uk.ac.ebi.eva.groovy.commons.RetryableBatchingCursor
+
+import static org.springframework.data.mongodb.core.query.Criteria.where
+import static uk.ac.ebi.eva.groovy.commons.EVADatabaseEnvironment.getDbsnpSvoeClass
+import static uk.ac.ebi.eva.groovy.commons.EVADatabaseEnvironment.getSvoeClass
+import static org.springframework.data.mongodb.core.query.Query.query
+
+class CorrectSSOperationFromEVA3399 {
+    static def logger = LoggerFactory.getLogger(CorrectSSOperationFromEVA3399.class)
+
+    EVADatabaseEnvironment prodEnv
+
+    CorrectSSOperationFromEVA3399() {}
+
+    CorrectSSOperationFromEVA3399(EVADatabaseEnvironment prodEnv) {
+        this.prodEnv = prodEnv
+    }
+
+    def correctSSOperations = {
+        def numRecordsProcessed = 0
+        def numRecordsUpdated = 0
+        [svoeClass, dbsnpSvoeClass].each { collectionClass ->
+            RetryableBatchingCursor cursor = new RetryableBatchingCursor<>(
+                    where("_id").regex('^EVA3399_'),
```

Comment:
```suggestion
                    where("_id").regex('^EVA3399_UPD_'),
```
Otherwise this also picks up SS [discard](https://github.com/EBIvariation/eva-tasks/blob/c77cef4bbed5bd693c3c4b28bab0d80f7c2583c9/tasks/eva_3399/src/main/groovy/eva3399/RemediateIndels.groovy#L150) and [merge ops](https://github.com/EBIvariation/eva-tasks/blob/c77cef4bbed5bd693c3c4b28bab0d80f7c2583c9/tasks/eva_3399/src/main/groovy/eva3399/RemediateIndels.groovy#L168) created in EVA3399.

It's probably not harmful to modify the reason for these (assuming these aren't also used in the release anywhere), but it would be misleading.